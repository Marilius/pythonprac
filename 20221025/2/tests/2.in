print(*list(slide('lolkek', 3)))