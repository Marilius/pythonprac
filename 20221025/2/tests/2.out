l o l o l k l k e k e k e k k